2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,37 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+
+const bottleFull = 100;
+const numOfUnits = 10;
+
 let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = bottleFull;
   },
+
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume !== bottleFull)
```

Comment:
This is correct! However, I'd recommend checking if the volume <= 100, to ensure that there is a no knock-on effect if the ``numOfUnits`` changed to some weird numbers like 15.